nfpm/shared_files/hornet.env

```diff
@@ -0,0 +1,5 @@
+# Use an alternative snapshot file location
+#REMOTE_FILE="http://my-alternative.location.io/file.bin.gz"
+
+# Add cli arguments to hornet
+#OPTIONS="--node.LogLevel=127 --useProfile=1gb"
```

Comment:
The `--node.LogLevel=127` cli arg got removed from HORNET (with HORNET v0.3.0).
Please add an alternative example (available cli args: `./hornet -h`) or remove it.